spec/rails_helper.rb

```diff
@@ -66,3 +66,12 @@
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
 end
+
+# Tests constants
+# API endpoints
+API_URL_PREFIX = '/api/v1'
+API_ENDPOINTS = {
+  Users: {
+    Signup: "#{API_URL_PREFIX}/users.json" # POST /.../users.json
+  }
+}.freeze
```

Comment:
Not sure this file is the right place for this

Reply:
Can we not access the standard rails routes in tests?

Reply:
You mean we should use xxx_path route helpers instead?
I thought about it but it felt like a problem because it violates the testing in isolation concept by depending on the route helpers which may not help us spot bugs.
I'm not sure if its the right place to do that there either, I haven't gave it a lot of thought but surely it felt right to have a single source of truth for endpoints to avoid duplication and errors on the run and it should be independent from anything (i.e. the actual routes) to spot issues.

Reply:
Do you want me to remove it?

Reply:
In this case, I think it makes sense to just use the routes since they already exist. I don't think it necessary violates the isolation concept, it just uses the predefined routes to know where to make the request to

Reply:
Alright, I'll use the route helpers instead.

Reply:
Done